pkg/controller/certificates/rootcacertpublisher/publisher.go

```diff
@@ -62,6 +62,12 @@ func NewPublisher(cmInformer coreinformers.ConfigMapInformer, cl clientset.Inter
 	e.cmLister = cmInformer.Lister()
 	e.cmListerSynced = cmInformer.Informer().HasSynced
 
+	nsInformer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
+		AddFunc:    e.namespaceAdded,
+		UpdateFunc: e.namespaceUpdated,
```

Comment:
Without registering these event handlers, namespaceAdded and namespaceUpdated are not called, so we don't resync the ca.crt config map on namespace changes.